packages/adblocker/src/filters/network.ts

```diff
@@ -153,6 +153,7 @@ export const enum NETWORK_FILTER_MASK {
   isHostnameAnchor = 1 << 28,
   isRedirectRule = 1 << 29,
   isRedirect = 1 << 30,
+  isTo = 1 << 31,
```

Comment:
why do we need the flag? looks like it is never read

it should not be needed if we convert `$to` to `$denyallow`

Reply:
This is required to build an original form in `toString`

Reply:
Not sure if this is good enough reason to use the slot in the mask